### PR TITLE
tests: fix role for aws govcloud

### PR DESCRIPTION
**Pull request by cpanato**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -251,6 +251,7 @@ pipeline {
       }
       environment {
         TECTONIC_INSTALLER_ROLE = 'tf-tectonic-installer-track-2'
+        TECTONIC_GOVCLOUD_INSTALLER_ROLE = 'tf-tectonic-installer'
```
Comment (enxebre) [inline Jenkinsfile]: should probably rather be named `TECTONIC_INSTALLER_GOVCLOUD_ROLE` wdyt?

Comment (enxebre) [reply Jenkinsfile]: doesn't the value need to be 'tf-tectonic-installer-govcloud'?

Comment (cpanato) [reply Jenkinsfile]: the role name in the govcloud accout is called `tf-tectonic-installer` :) 

I will update the first comment thanks!

**Comment by cpanato** [comment]
@enxebre PTAL :)

**Comment by cpanato** [comment]
this is not needed anymore

**Pull request closed without merging**